discord/ui/input_text.py

```diff
@@ -236,3 +247,12 @@ def to_component_dict(self) -> InputTextComponentPayload:
 
     def refresh_state(self, data) -> None:
         self._input_value = data["value"]
+
+    def refresh_from_modal(self, interaction: Interaction, data: dict) -> None:
+        return self.refresh_state(data)
+
+    def uses_label(self) -> bool:
+        return self.description is not None
+
+
+TextInput = InputText
```

Comment:
Yay!

Reply:
yeah i figured we should at the very least have this alias... no idea how the name was decided originally